packages/subgraph/src/mappings/order.ts

```diff
@@ -26,7 +34,25 @@ export function handleDCAOrderInitialized(event: Initialized): void {
   order.sellToken = createOrReturnTokenEntity(orderContract.sellToken()).id;
   order.buyToken = createOrReturnTokenEntity(orderContract.buyToken()).id;
   order.receiver = orderContract.receiver();
+
+  let protocolFee: BigInt = BigInt.fromI32(0);
+  let orderFactoryAddress: Address = (
+    event.transaction.to !== null
+      ? event.transaction.to
+      : Address.fromString("0x0")
+  )!;
+
+  const factory = OrderFactory.bind(orderFactoryAddress);
+  let tryProtocolFee = factory.try_protocolFee();
+  if (!tryProtocolFee.reverted) {
+    protocolFee = BigInt.fromI32(tryProtocolFee.value);
+  }
+
   order.amount = orderContract.amount();
+  order.fee = protocolFee;
+  order.feeAmount = order.amount
+    .times(protocolFee)
+    .div(HUNDRED_PERCENT.minus(protocolFee));
```

Comment:
is this related to adding address to `Token`?

Reply:
There are 2 commits in this PR. One to update the code to the last changes on Stackly repo another to add the address to token entity.